ci/dockerfiles/ansible-e2e-hybrid.Dockerfile

```diff
@@ -43,6 +43,10 @@ COPY --from=builder /go/src/github.com/operator-framework/operator-sdk/bin/ao-lo
 COPY --from=builder /ansible/memcached-operator/watches.yaml ${HOME}/watches.yaml
 COPY --from=builder /ansible/memcached-operator/roles/ ${HOME}/roles/
 
+
+RUN find ${HOME} -type f -exec chmod -R g+rw {} \;  && \
+    find ${HOME} -type d -exec chmod -R g+rwx {} \;
```

Comment:
Need chmod for `/usr/local/bin` and `/usr/share/ansible/openshift` here as well for the COPYs in lines 40-42.